controllers/storagecluster/reconcile.go

```diff
@@ -87,11 +87,39 @@ osd_memory_target_cgroup_limit_ratio = 0.5
 
 var storageClusterFinalizer = "storagecluster.ocs.openshift.io"
 
-var validTopologyLabelKeys = []string{
-	"failure-domain.beta.kubernetes.io",
-	"failure-domain.kubernetes.io",
-	"kubernetes.io/hostname",
-	"topology.rook.io",
+const labelZoneRegionWithoutBeta = "failure-domain.kubernetes.io/region"
+const labelZoneFailureDomainWithoutBeta = "failure-domain.kubernetes.io/zone"
+const labelRookPrefix = "topology.rook.io"
+
+func getValidTopologyLabelKeys() []string {
+
+	var s []string
+
+	s = append(s,
```

Comment:
instead of append should not we create a slice here itself like `s := []string{"a", "b", "c", "d"}`

Reply:
Choosing to create using append to keep future diffs cleaner.